### PR TITLE
Readme.md updated

### PR DESCRIPTION
**Pull request by balwanishivam**

### CODE REVIEW

**Comment by balwanishivam** [comment]
Tried to Update Readme.md

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+$ pip install -r requirements.txt
+$ python manage.py runserver
+```
+### To make migrations
+```sh
+$ python manage.py makemigrations
+$ python manage.py migrate
+```
+### Author :pencil:
+©️ [balwanishivam](https://github.com/balwanishivam) :tada:
```
Comment (Monal5031) [inline README.md:39]: 🤨

Comment (balwanishivam) [reply README.md:39]: Sorry i forgot to remove author i will make these changes

Comment (balwanishivam) [reply README.md]: Changes made

**Pull request merged**